### DISCUSSION TITLE
May missing files in folder core-main-data

### DISCUSSION THREAD

**Discussion by Haibao233** [dcm4che/dcm4che#921] [Q&A] [answered]
I downloaded the whole project and ran it found missing StandardElementDictionary.java and Keyword.java in dcm4che-core/src/main/java/org/dcm4che3/data/.

you can find them in 
https://github.com/dcm4che/dcm4che/commit/0ee60838003e55851838af1244fb980a97582953
We can fix the problem by making up for them(Here's how I solve the problem)

**Comment by gunterze** [comment, marked as answer]
Did you run
```
mvn install
```
?
s. https://github.com/dcm4che/dcm4che#build

**Reply by Haibao233** [reply]
sorry，that's my fault。Idea help me install some jar that make me ignore your Build tip。
